### DISCUSSION TITLE
续费专业版，订单查询显示ip不在白名单，专业版时长没有增加

### DISCUSSION THREAD

**Discussion by Barb-Y** [getlantern/lantern-forum-cn#914] [Q&A]
支付宝付款，收款方：广州饶铭网络科技有限公司，订单号：2024100122001400491412397404，商家订单号：2024100110571623753，支付时间：2024-10-01 10:58:27

![_cgi-bin_mmwebwx-bin_webwxgetmsgimg__ MsgID=4182412597293183324 skey=@crypt_f9927ec_ed5c8060f3b8364d06634184924c7053 mmweb_appid=wx_webfilehelper](https://github.com/user-attachments/assets/dcfcc25f-170d-4de5-be07-7664434dbb05)

**Comment by MaxMKMax** [comment]
@Barb-Y 您好. 请您也提供我账号邮箱. 我帮您提交问题上去.

**Comment by Barb-Y** [comment]
您好. ***@***.***


谢谢.




---- 回复的原邮件 ----
| 发件人 | Mike ***@***.***> |
| 发送日期 | 2024年10月02日 02:19 |
| 收件人 | getlantern/lantern-forum-cn ***@***.***> |
| 抄送人 | Barb-Y ***@***.***>,
Mention ***@***.***> |
| 主题 | Re: [getlantern/lantern-forum-cn] 续费专业版，订单查询显示ip不在白名单，专业版时长没有增加 (Discussion #914) |

@Barb-Y 您好. 请您也提供我账号邮箱. 我帮您提交问题上去.

**Comment by MaxMKMax** [comment]
@Barb-Y 
您好. 这是您的激活码:
VD39K-VHCRP-B32W4-7Y7MT-8FVB6

**Comment by Barb-Y** [comment]
您好！我买的是2年续期，应该是2027/3/10到期。但输入此激活码后只给了1年续期，麻烦处理，谢谢！

---- 回复的原邮件 ----
| 发件人 | Mike ***@***.***> |
| 发送日期 | 2024年10月24日 00:42 |
| 收件人 | getlantern/lantern-forum-cn ***@***.***> |
| 抄送人 | Barb-Y ***@***.***>,
Mention ***@***.***> |
| 主题 | Re: [getlantern/lantern-forum-cn] 续费专业版，订单查询显示ip不在白名单，专业版时长没有增加 (Discussion #914) |

@Barb-Y
您好. 这是您的激活码:
VD39K-VHCRP-B32W4-7Y7MT-8FVB6

**Comment by MaxMKMax** [comment]
@Barb-Y 您好. 我已经帮您提交问题上去. 请您稍等几天到 1 个星期. 支付部门会帮您解决的.

**Comment by MaxMKMax** [comment]
@Barb-Y 请您在此提供我账号邮箱. 直接截图账号邮箱给我吧. 被Github屏蔽了. 看不到. 谢谢!

**Comment by Barb-Y** [comment]
***@***.***

---- 回复的原邮件 ----
| 发件人 | Mike ***@***.***> |
| 发送日期 | 2024年10月26日 23:14 |
| 收件人 | getlantern/lantern-forum-cn ***@***.***> |
| 抄送人 | Barb-Y ***@***.***>,
Mention ***@***.***> |
| 主题 | Re: [getlantern/lantern-forum-cn] 续费专业版，订单查询显示ip不在白名单，专业版时长没有增加 (Discussion #914) |

@Barb-Y 请您在此提供我账号邮箱. 直接截图账号邮箱给我吧. 被Github屏蔽了. 看不到. 谢谢!

**Comment by Barb-Y** [comment]
谢谢

---- 回复的原邮件 ----
| 发件人 | Mike ***@***.***> |
| 发送日期 | 2024年10月26日 23:14 |
| 收件人 | getlantern/lantern-forum-cn ***@***.***> |
| 抄送人 | Barb-Y ***@***.***>,
Mention ***@***.***> |
| 主题 | Re: [getlantern/lantern-forum-cn] 续费专业版，订单查询显示ip不在白名单，专业版时长没有增加 (Discussion #914) |

@Barb-Y 请您在此提供我账号邮箱. 直接截图账号邮箱给我吧. 被Github屏蔽了. 看不到. 谢谢!

**Comment by MaxMKMax** [comment]
@Barb-Y 您好. 非常抱歉, 依然看不到. 要不您这么写吧. babyatqqdotcom  用(at) (dot) 用代替 @ . 
这样我就能看到了.

**Comment by Barb-Y** [comment]
您好
liaowuqiangua2019at163dotcom
谢谢！

---- 回复的原邮件 ----
| 发件人 | Mike ***@***.***> |
| 发送日期 | 2024年10月29日 00:34 |
| 收件人 | getlantern/lantern-forum-cn ***@***.***> |
| 抄送人 | Barb-Y ***@***.***>,
Mention ***@***.***> |
| 主题 | Re: [getlantern/lantern-forum-cn] 续费专业版，订单查询显示ip不在白名单，专业版时长没有增加 (Discussion #914) |

@Barb-Y 您好. 非常抱歉, 依然看不到. 要不您这么写吧. babyatqqdotcom 用(at) (dot) 用代替 @ .
这样我就能看到了.
